src/App.tsx

```diff
@@ -29,18 +20,16 @@ function App() {
               <SignIn />
             </Container>
           </Route>
-          <Route exact path="/">
-            <Container component="main" className={classes.main} maxWidth="sm">
-              <Main />
-            </Container>
-          </Route>
           <Route path="/dashboard">
             <Dashboard />
           </Route>
+          <Route path="/">
+            <Redirect from="/" to={auth.authenticated ? "/dashboard" : "/login"} />
+          </Route>
```

Comment:
```suggestion
          <Redirect to={auth.authenticated ? "/dashboard" : "/login"} />
```

Siden `react-router` sjekker fra topp til bunn holder det med en `Redirect` her nede :)